woocommerce/woocommerce-android
# Issue/6585 onboarding SL banner UI

Part of: #6585 


### Description
This PR adds: 

- New "Install WC Shipping" banner UI to order detail
- A feature flag that ensured the banner is only displayed for debug builds

### Testing instructions
Open order detail and check the banner is displayed in `debug` build. Banner UI should match this design 9PGX5zb3VKHMYc7qTIv30B-fi-744%3A26803

### Images


<img width="300"  src="https://user-images.githubusercontent.com/2663464/170314781-9abc5012-e534-44fc-9852-4768aaeed3dc.png">
